susemanager-utils/susemanager-sls/salt/ssh_bootstrap/init.sls

```diff
@@ -61,5 +61,18 @@ authorize_own_key:
       - file: ownership_own_ssh_key
       - ssh_auth: no_own_key_authorized
 
+logrotate_configuration:
+  file.managed:
+    - name: /etc/logrotate.d/salt-ssh
```

Comment:
- user: root
    - group: root
    - mode: 644

Reply:
That's already the case, as you can see above. But OK, it doesn't hurt having it.